frontend/test/__support__/server-mocks/activity.ts

```diff
@@ -10,14 +10,13 @@ export function setupRecentViewsEndpoints(recentItems: RecentItem[]) {
 
 export function setupRecentViewsAndSelectionsEndpoints(
   recentItems: RecentItem[],
+  context: string = "context=selections&context=views",
```

Comment:
Right now this argument is a _query_, not a _context_.

It'd be great if this function accepted `context: RecentContexts[]` and used `querystring` to generate the URL for us.

```suggestion
  context: RecentContexts[] = ["selections", "views"],
```